src/configure/configurers/localGithubWorkflowConfigurer.ts

```diff
@@ -9,32 +9,75 @@ import { UserCancelledError } from 'vscode-azureextensionui';
 import * as constants from '../resources/constants';
 import { Messages } from '../resources/messages';
 import { TelemetryKeys } from '../resources/telemetryKeys';
 import { TracePoints } from '../resources/tracePoints';
 import { Configurer } from "./configurerBase";
 
-const uuid = require('uuid/v4');
 
+const uuid = require('uuid/v4');
 const Layer = 'LocalGitHubWorkflowConfigurer';
 
 export class LocalGitHubWorkflowConfigurer implements Configurer {
     protected githubClient: GithubClient;
-    private queuedPipelineUrl: string;;
+    private queuedPipelineUrl: string;
+    private controlProvider: ControlProvider;
+    private localGitRepoHelper : LocalGitRepoHelper;
 
-    constructor(azureSession: AzureSession, subscriptionId: string) {
+    constructor(azureSession: AzureSession, subscriptionId: string, localgitRepoHelper: LocalGitRepoHelper) {
+        this.controlProvider = new ControlProvider();
+        this.localGitRepoHelper = localgitRepoHelper;
     }
 
     public async getInputs(inputs: WizardInputs): Promise<void> {
         this.githubClient = new GithubClient(inputs.githubPATToken, inputs.sourceRepository.remoteUrl);
-        return;
+        try {
+            inputs.isNewOrganization = false;
+            if (!inputs.sourceRepository.remoteUrl) {
+                let githubOrganizations = await this.githubClient.listOrganizations(inputs.githubPATToken);
+               
+                if ( githubOrganizations &&  githubOrganizations.length > 0) {
+                    let selectedOrganization = await this.controlProvider.showQuickPick(
+                        constants.SelectGitHubOrganization,
+                        githubOrganizations.map(x => { return { label: x.login }; }),
+                        { placeHolder: Messages.selectGitHubOrganizationName },
+                        TelemetryKeys.OrganizationListCount);
+                        inputs.organizationName = selectedOrganization.label;
+
+
+                    let newGitHubRepo = await this.githubClient.generateGitHubRepository(inputs.organizationName, inputs.sourceRepository.localPath) as unknown as GitHubRepo | void;
+                    if(newGitHubRepo){
+                        inputs.sourceRepository.remoteName = newGitHubRepo.name;
+                        inputs.sourceRepository.remoteUrl = newGitHubRepo.html_url+".git";
+                        inputs.sourceRepository.repositoryId = GitHubProvider.getRepositoryIdFromUrl(inputs.sourceRepository.remoteUrl);
+                        await this.localGitRepoHelper.initializeGitRepository(inputs.sourceRepository.remoteName, inputs.sourceRepository.remoteUrl);
+                        vscode.window.showInformationMessage(utils.format(Messages.newGitHubRepositoryCreated, newGitHubRepo.name));
+                    }
+                    else{
+                        vscode.window.showErrorMessage(Messages.cannotCreateGitHubRepository);
+                        throw Error;
```

Comment:
throwing empty error is not very good as it will not help while analyzing the failures. You can throw same error messages as what you are showing to user.

Reply:
Okay, will change it.